packages/components/app/styles/components/pagination.scss

```diff
@@ -31,22 +31,47 @@ $pagination-layout-breakpoint: 1000px;
     flex-wrap: wrap;
     justify-content: center;
   }
+
+  // Apply group layout styles to child components only when used inside pagination wrapper:
+  .hds-pagination-nav {
+    grid-area: nav;
+
+    @media screen and (max-width: $pagination-layout-breakpoint) {
+      justify-content: center;
+      order: -1;
+      // Force nav to occupy a full row so other components will wrap:
+      width: 100%;
+    }
+  }
+
+  .hds-pagination-size-selector {
+    grid-area: selector;
+    justify-self: flex-end;
+    margin-left: var(--token-pagination-child-spacing-horizontal);
+
+    @media screen and (max-width: $pagination-layout-breakpoint) {
+      margin-top: var(--token-pagination-child-spacing-vertical);
+      margin-right: var(--token-pagination-child-spacing-horizontal);
+    }
+  }
+
+  .hds-pagination-info {
+    grid-area: info;
+    justify-self: flex-start;
+    margin-right: var(--token-pagination-child-spacing-horizontal);
+
+    @media screen and (max-width: $pagination-layout-breakpoint) {
+      margin-top: var(--token-pagination-child-spacing-vertical);
+      margin-left: var(--token-pagination-child-spacing-horizontal);
+    }
```

Comment:
Originally I was going to just move the margin styles to apply to nested child components vs. stand-alone but I realized other layout styles related to grid-area etc. also should be moved.

Reply:
I think is the right choice. It's even more readable now! :)